gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -868,7 +870,7 @@ end
 -- @return boolean
 -- @realm shared
 function plymeta:IsReady()
-	return self.is_ready or false
```

Comment:
Wouldn't this always be true, as the variable name is the exact same name as the function? And thus, the function or the variable will be overridden?

Reply:
`isReady` != `IsReady`